spec/robots/update_marc_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+describe Robots::DorRepo::Release::UpdateMarc do
+
+  before :all do
+    Dor::Config.release.write_marc_script = 'bin/write_marc_record_test'
+    Dor::Config.release.symphony_path = './spec/fixtures/sdr-purl'
+    Dor::Config.release.purl_base_uri = "http://purl.stanford.edu"
+  end
+
+  before :each do
+    @druid='aa222cc3333'
+    @umr = Robots::DorRepo::Release::UpdateMarc.new 
+  end  
+
+  it "runs the robot for a druid without a catkey and do nothing as a result" do
```

Comment:
i'd probably change "runs" to "run".  or "do" to "does"?  just so the part in quotes reads as a nice consistent sentence, regardless of whether the "it" command reads as part of that sentence (i've got no personal preference on that part since i don't find DSLs like this to read very naturally anyway).